wagtail/images/tests/test_image_operations.py

```diff
@@ -771,7 +876,7 @@ def test_default_quality(self):
         with patch("PIL.Image.Image.save") as save:
             fil.run(image, f)
 
-        save.assert_called_with(f, "WEBP", quality=85, lossless=False)
+        save.assert_called_with(f, "WEBP", quality=80, lossless=False)
```

Comment:
this will fail because https://github.com/wagtail/wagtail/pull/10657/files#diff-c195bc8bd4d5a11ae02d0451edb4dd4a8e7b0014398a384a1be3ff3c355a5ee8R979 is still 85

Reply:
yes I noticed, will push the change with docs